ra2ce/network/networks.py

```diff
@@ -154,6 +156,26 @@ def _export_network_files(
         self.graph_files.set_graph(graph_type, network)
 
     def _get_new_network_and_graph(self, export_types: list[str]) -> None:
+        def _get_edges_time_hours(_graph: nx.MultiGraph) -> dict:
```

Comment:
You are assuming that `avgspeed` will be present for all those dataframes. This makes the whole test bench fail. Please modify the code below so it avoids it.

Reply:
I checked and here e[-1].get("avgspeed") returns None if avgspeed does not exist. I tested it. Where does the issue come from?